jobs/worker.go

```diff
@@ -91,3 +98,45 @@ func RunJobNow(j Job) {
 	}
 	l.Log.Infof("Finished Job %v with %v", j.Name(), j.Arguments())
 }
+
+// healthCheckStruct is a struct that contains the time for the health checker and a channel to pass the shared memory for the time through time channel
+type healthCheckStruct struct {
+	timeStamp   time.Time
+	timeChannel chan time.Time
+}
+
+func (hc *healthCheckStruct) healthChecker() {
+	//run healthcheck loop and consumer in the background
+	go hc.healthcheckLoop()
+	go hc.healthCheckConsumer()
+
+	//create new GET request through echo with the same path name "/health"
+	e := echo.New()
```

Comment:
Can we also tack
```go
	e.HideBanner = true
	e.HidePort = true
```
After this? So we don't get the ASCII art output to keep our logs pure JSON.